Sources/Plasma/PubUtilLib/plWinDpi/plWinDpi.h

```diff
@@ -47,10 +47,29 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include <variant>
 
 #include "hsWindows.h"
-#include <ShellScalingApi.h>
+#if __has_include(<ShellScalingApi.h>)
+#   include <ShellScalingApi.h>
+#endif
 
 #include "plStatusLog/plStatusLog.h"
 
+#ifndef DPI_ENUMS_DECLARED
```

Comment:
Is this defined in `ShellScalingApi.h`?

Reply:
Yes, I don't love using something that's not documented API, but MinGW doesn't ship the ShellScalingApi header.

I could alternatively define these in the `#else` block of the `#if __has_include` if you prefer that

Reply:
Yeah, I feel like that might be a little more obvious.